### ISSUE TITLE
Add composer repository

### ISSUE THREAD

**Issue by timiwahalahti** [mustardBees/cmb_field_map#41]
Hi,

Since this plugin isn't in w.org plugin repository, it can't be used with composer and wpackagist.org mirror. Composer repo would be awesome, and it's simple to made with [packagist](https://packagist.org/).

**Comment by timiwahalahti** [comment]
Yeah, so. The repo contains composer.json but this repository is not in packagist so it does not work. You should submit this repo into packagist in favor of making it work.

**Comment by rasmustaarnby** [comment]
@timiwahalahti  just stumbled upon this. If you add the following to your repositories in your composer file: 
```
{
  "type": "vcs",
  "url": "https://github.com/mustardBees/cmb_field_map.git"
}
```
Then in the require section you add:

`"mustardbees/cmb_field_map": "2.1.2",`

Then you don't need it to be on wpackagist.org
